stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -10,40 +10,86 @@ This implementation supports introducing all WebSub components: subscribers, pub
 is updated by the topic's publisher.
  
  
-#### Basic Flow with WebSub
+### Basic Flow with WebSub
 1. Subscriber discovers the topics it needs to subscribe to in order to receive updates/content and discovers the hub(s) where it can subscribe.
```

Comment:
```suggestion
1. The subscriber discovers the topics it needs to subscribe to in order to receive updates/content and discovers the hub(s) to which it can subscribe.
```

Reply:
Changed the former, but re: the latter, you subscribe **to a topic** at (a) particular hub(s), which is why "where" was used. Will change to "at which" WDYT?

Reply:
We don't subscribe "at a topic". It should be "to a topic".

Reply:
Yeah exactly, at is used for the hub here.

Subscribing **to** a topic **at** a hub.

Reply:
Changing to " The subscriber discovers from the publisher, the topic it needs to subscribe to and the hub(s) that deliver content on updates of the topic." as discussed offline.